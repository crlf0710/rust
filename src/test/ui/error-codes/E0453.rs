// check-pass
#![forbid(non_snake_case)]

#[allow(non_snake_case)]
//~^ WARNING allow(non_snake_case) overruled by outer forbid(non_snake_case)
fn main() {
}
