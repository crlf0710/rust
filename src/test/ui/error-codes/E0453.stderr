warning[E0453]: allow(non_snake_case) overruled by outer forbid(non_snake_case)
  --> $DIR/E0453.rs:4:9
   |
LL | #![forbid(non_snake_case)]
   |           -------------- `forbid` level set here
LL | 
LL | #[allow(non_snake_case)]
   |         ^^^^^^^^^^^^^^ overruled by previous forbid

warning: 1 warning emitted

For more information about this error, try `rustc --explain E0453`.
